graphql/types/Logistics.graphql

```diff
@@ -17,6 +17,7 @@ type PickupPoint {
   """
   If false, it means pickup point is currently disabled
   """
+  pickupHolidays: [HolidayHours]
```

Comment:
This should be below the `isActive` because of the comment above. If possible, add a comment for this field, also.